linkerd/proxy/discover/src/from_resolve.rs

```diff
@@ -97,29 +98,36 @@ impl<R: TryStream, E> Discover<R, E> {
 impl<R, E> Stream for Discover<R, E>
 where
     R: TryStream<Ok = Update<E>>,
-    E: Clone,
+    E: Clone + std::fmt::Debug,
 {
     type Item = Result<Change<SocketAddr, E>, R::Error>;
 
     fn poll_next(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
         loop {
             let this = self.as_mut().project();
             if let Some(change) = this.pending.pop_front() {
+                trace!(?change, "Changed");
                 return Poll::Ready(Some(Ok(change)));
             }
 
+            trace!("poll");
```

Comment:
i'm assuming we're inside a span that would give more context about what's being polled, here? so this won't just spam "poll" a bunch?